pkgs/development/libraries/hunspell/dictionaries.nix

```diff
@@ -48,11 +48,11 @@ let
       nativeBuildInputs = [ bash coreutils which zip unzip ];
       patchPhase = ''
```

Comment:
This could use postPatch otherwise patches does not work

Reply:
Do you mean moving the code there to `postPatch`?